docs/Options.md

```diff
@@ -180,86 +239,69 @@ Values: `true` or `false` | Default =  `false`
 
-Values: `String` | Default =  `localhost`
+Example: `LD_SINGLEFILE_OPTIONS=--user-agent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:124.0) Gecko/20100101 Firefox/124.0"`
 
-The hostname or IP of the database server.
 
-### `LD_DB_PORT`
+### `LD_SINGLEFILE_PATH`
+Values: `String` | Default = "single-file"
 
-Values: `Integer` | Default =  None
+Path to the single-file executable. Defaults to `single-file` which is stored on the path. This option would rarely need to be updated.
 
-The port of the database server.
-Should use the default port if left empty, for example `5432` for PostgresSQL.
 
-### `LD_DB_OPTIONS`
+### `LD_SINGLEFILE_TIMEOUT_SEC`
 
-Values: `String` | Default = `{}`
+Values: `Float` | Default = 120.0
 
-A json string with additional options for the database. Passed directly to OPTIONS.
+When creating HTML archive snapshots, control the timeout for how long to wait for the snapshot to complete, in `seconds`.
+Defaults to 60 seconds; on lower-powered hardware you may need to increase this value. 
 
-### `LD_FAVICON_PROVIDER`
+### `LD_SINGLEFILE_UBLOCK_OPTIONS`
 
-Values: `String` | Default =  `https://t1.gstatic.com/faviconV2?client=SOCIAL&type=FAVICON&fallback_opts=TYPE,SIZE,URL&url={url}&size=32`
+Values: `String` | Default = various
 
-The favicon provider used for downloading icons if they are enabled in the user profile settings.
-The default provider is a Google service that automatically detects the correct favicon for a website, and provides icons in consistent image format (PNG) and in a consistent image size.
+When a single file is created, the headless browser it uses also utilises the ublock plugin to remove cookie popup and some ads. The executable can be passed a number of flags to change how this plugin works. This option can be overridden to provide those flags into the executable.
 
-This setting allows to configure a custom provider in form of a URL.
-When calling the provider with the URL of a website, it must return the image data for the favicon of that website.
-The configured favicon provider URL must contain a placeholder that will be replaced with the URL of the website for which to download the favicon.
-The available placeholders are:
-- `{url}` - Includes the scheme and hostname of the website, for example `https://example.com`
-- `{domain}` - Includes only the hostname of the website, for example `example.com`
+The default flags passed into single-file are:
 
-Which placeholder you need to use depends on the respective favicon provider, please check their documentation or usage examples.
-See the default URL for how to insert the placeholder to the favicon provider URL.
+```
+--browser-arg="--headless=new"
+--browser-arg="--user-data-dir=./chromium-profile"
+--browser-arg="--no-sandbox"
+--browser-arg="--load-extension=uBlock0.chromium"
+```
 
-Alternative favicon providers:
-- DuckDuckGo: `https://icons.duckduckgo.com/ip3/{domain}.ico`
+## Superuser Options
```

Comment:
These might be the most used options, which is why I had it at the top. Let's move it back there.